sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/implementation/SerializationUtil.java

```diff
@@ -21,18 +18,15 @@
      * @param mapper the mapper to be configured
      */
     public static void configureMapper(ObjectMapper mapper) {
-        SimpleDateFormat df = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'");
-        df.setTimeZone(TimeZone.getDefault());
-        mapper.setDateFormat(df);
-
-        mapper.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
+        mapper.registerModule(new JavaTimeModule());
         mapper.disable(DeserializationFeature.ADJUST_DATES_TO_CONTEXT_TIME_ZONE);
 
         UntypedObjectDeserializer defaultDeserializer = new UntypedObjectDeserializer(null, null);
-        Iso8601DateDeserializer dateDeserializer = new Iso8601DateDeserializer(defaultDeserializer);
-        GeoPointDeserializer geoPointDeserializer = new GeoPointDeserializer(dateDeserializer);
+        GeoPointDeserializer geoPointDeserializer = new GeoPointDeserializer(defaultDeserializer);
+        Iso8601DateDeserializer iso8601DateDeserializer = new Iso8601DateDeserializer(geoPointDeserializer);
         SimpleModule module = new SimpleModule();
-        module.addDeserializer(Object.class, geoPointDeserializer);
+        module.addDeserializer(Object.class, iso8601DateDeserializer);
+        mapper.registerModule(Iso8601DateSerializer.getModule());
```

Comment:
Is this required? `module`, which has the date deserializer, is registered to the `mapper` in the next line.

Reply:
Here we register both serializer and deserializer for java.utl.Date type